### PR TITLE
Reuse pino instance if passed with 'logger' option. Document the serializers

### LINKED ISSUE #2: support custom pino instance.
Reported as https://github.com/mcollina/express-pino-logger/issues/6.

Depends on https://github.com/mcollina/pino/issues/66

### PR DESCRIPTION
**Pull request by mcollina**
Fixes #2 #7 

cc @marcbachmann @davidmarkclements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,10 +89,98 @@ $ node example.js | pino
...
+'use strict'
+
+var pino = require('pino-http')
+var logger = pino({
+  serializers: {
+    req: pino.stdSerializers.req,
+    res: pino.stdSerializers.res
+  }
+})
+
```
Comment (marcbachmann) [inline README.md]: There's a bit much red. Looks like you forgot the backticks here.

File: logger.js [hunk 2]
```diff
@@ -14,7 +14,15 @@ function pinoLogger (opts, stream) {
   opts.serializers.req = opts.serializers.req || asReqValue
   opts.serializers.res = opts.serializers.res || pino.stdSerializers.res
 
-  var logger = pino(opts, stream)
+  var logger = null
+
+  if (opts.logger) {
+    logger = opts.logger
+    opts.logger = undefined
```
Comment (marcbachmann) [inline logger.js]: Please don't modify the properties of a foreign object.

File: README.md (hunk 1 above)
Comment (mcollina) [inline README.md]: Done :)

File: logger.js (hunk 2 above)
Comment (mcollina) [inline logger.js]: Should be good now, check :)

**Comment by gabrieleds** [comment]
👍 Yep, just what I was looking for ;-)

**Comment by mcollina** [comment]
@davidmarkclements merging and releasing.

**Pull request merged**